src/pathOrderOptimizer.cpp

```diff
@@ -260,7 +260,7 @@ void LineOrderOptimizer::optimize(bool find_chains)
                 num_joined_lines[point_idx] = 0;
                 const Point& p = (*polygons[poly_idx])[point_idx];
                 // look at each of the lines that finish close to this line to see if either of its vertices are coincident this vertex
-                for (unsigned int close_line_idx : line_bucket_grid.getNearbyVals(p, gridSize))
+                for (unsigned int close_line_idx : line_bucket_grid.getNearbyVals(p, 10))
```

Comment:
The number 10, what was the reason to change it?

Reply:
There, we're only interested in finding points that are coincident and so there is no  benefit in asking for points that are further away than 10 um. I believe that getNearbyVals() returns points that can be as far away as gridSize+queryRadius so it's probably still returning points that are up to 2mm away when we only want points that are a few um away.